app/assets/javascripts/listings/components/features-section.html.slim

```diff
@@ -14,6 +14,8 @@ accordion-heading
     feature-row title="Appliances" description="{{::$ctrl.parent.listing.Appliances}}"
     feature-row title="{{$ctrl.parent.isOwnershipListing($ctrl.parent.listing) ? 'Services Covered by HOA Dues' : 'Services Onsite' }}" description="{{::$ctrl.parent.listing.Services_Onsite}}"
     feature-row title="Parking" description="{{::$ctrl.parent.listing.Parking_Information}}"
+    feature-row title="Smoking Policy" description="{{::$ctrl.parent.listing.Smoking_Policy}}"
```

Comment:
why not use locale-en.json for this if we already have it defined?

Reply:
I wasn't 100% sure how to integrate the use of `translate` in there, and google translate runs through this page anyway. But I can try to see if if something like `title="{{SMOKING_POLICY | translate }}"` will work!